source/guide_goaccess.rst

```diff
@@ -0,0 +1,199 @@
+
+ * ``--prefix=/home/isabell``: Installation target for your personal Uberspace.
+ * ``--enable-utf8``: Compile with wide character support.
+ * ``--enable-geoip=legacy``: Compile with GeoLocation support, legacy will utilize the original GeoIP databases.
+
+Other options can be found in the GoAccess `installation documentation`_.
+
+.. warning:: Replace ``isabell`` with your Uberspace name!
+
+::
+
+ [isabell@stardust ~]$ cd ~/filebase/goaccess/
+ [isabell@stardust ~]$ ./configure --enable-utf8 --enable-geoip=legacy --prefix=/home/isabell
+ [isabell@stardust ~]$ make
+ [isabell@stardust ~]$ make install
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+After the installation of GoAccess, it is necessary to enable the web server logs and to configure GoAccess with the right log format.
+
+Step 1 - Enable the Web Server Log
+----------------------------------
+
+Please follow the instructions by the Uberspace manual_ to enable the web server logs.
+
+.. note:: Please consider, after the web server log enabling, it needs some time to have some entries in your new log files. Depends on the web traffic.
+
+Step 2 - GoAccess Configuration
+-------------------------------
+
+Edit the configuration file ``~/etc/goaccess/goaccess.conf`` and reach out for the following parameters to uncomment these.
+
+.. code-block:: bash
+ :emphasize-lines: 2,4,6,8
+ 
+ # Time Format Options (required)
+ time-format %H:%M:%S
+ 
+ # Date Format Options (required)
+ date-format %d/%b/%Y
+ 
+ # NCSA Combined Log Format (is in use by Uberspace)
+ log-format %h %^[%d:%t %^] "%r" %s %b "%R" "%u"
+ 
+ # Set HTML report page title and header.
+ html-report-title My Uberspace
+
+Launch
+======
+
+Step 1 - First Try (or Realtime Analysis in the Shell)
+------------------------------------------------------
+
+To get first results, to check that everthing is maintained, please enter:
+
+::
+
+ [isabell@stardust ~]$ goaccess -a -p ~/etc/goaccess/goaccess.conf -f ~/logs/webserver/access_log
+
+Scroll with your cursor keys up and down. With "q" you can quit GoAccess.
+
+The used parameters are:
+
+ * ``-a``: Enable a list of user-agents by hosts.
+ * ``-p``: Config file location.
+ * ``-f``: Web server log file location.
+
+Step 2 - HTML Output
+--------------------
+
+This is the graphical variant as static HTML web page. I consider a new folder inside the html location, that you have access from outside on your statistics.
+
+::
+
+ [isabell@stardust ~]$ cd ~/html/
+ [isabell@stardust ~]$ mkdir statistics
+ [isabell@stardust ~]$
+
+The command to create a static file with GoAccess is:
+
+::
+
+ [isabell@stardust ~]$ goaccess -a -p ~/etc/goaccess/goaccess.conf -f ~/logs/webserver/access_log -o ~/html/statistics/report.html
+
+The additional parameter is:
+
+ * ``-o``: To define the output file.
+
+.. note:: The target folder ``statistics`` and file name ``report.html`` are examples. If you want more privacy after this publication, please use other names for boths.
```

Comment:
Using an "other name" isn't a good practice for access control. How about basic auth inside a `.htaccess` file?